mandatory/1-fix-functions.js

```diff
@@ -32,7 +30,7 @@ function greaterThan10(num) {
 
 function get3rdIndex(arr) {
   let index = 3;
-  let element;
+  let element= arr[index]
```

Comment:
Sometimes writing too much code can be unnecessary and takes up too much space. so you can just do
 let element= arr[3];
return element;